using-discord-bot.py

```diff
@@ -0,0 +1,10 @@
+import discord
+from discord.ext import commands
+
+bot = discord.Bot(command_prefix='!', help_command=None)
+
```

Comment:
There's no `discord.Bot` class in the discord.py library. However, I did find `commands.Bot` class in it which is what I'm assuming you're trying to do here. Kindly fix that and I'll approve the pull request.

Reply:
aaah I did it wrong its command.Bot. ;-